processing/src/main/java/org/apache/druid/frame/processor/OutputChannelFactory.java

```diff
@@ -31,6 +31,11 @@
    */
   OutputChannel openChannel(int partitionNumber) throws IOException;
 
+  /**
+   * Create a channel pair tagged with a particular name and a flag to delete the channel data after its read.
+   */
+  PartitionedOutputChannel openChannel(String name, boolean deleteAfterRead) throws IOException;
```

Comment:
Would be clearer as `openPartitionedChannel`.

Reply:
done, thanks for the suggestion!